ci/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")/.."
+
+spatial build --log_level debug
```

Comment:
This is inconsistent with the other pr (which puts `--log_level` on a new line).

Reply:
Updated the other to be consistent with this 😄